src/web/app.js

```diff
@@ -653,6 +657,25 @@ BF2042Portal.Extensions = (function () {
             return promise;
         }
 
+        const saveTemplateAsFile = (filename, dataObjToWrite) => {
+            const blob = new Blob([JSON.stringify(dataObjToWrite, null, 4)], { type: "text/json" });

```

Comment:
Due to the json structure of blockly an indent of 2 or even only 1 would probably be better